src/mixins/filing-template-mixin.ts

```diff
@@ -1393,8 +1394,11 @@ export default class FilingTemplateMixin extends DateMixin {
     this.setEntityType(entitySnapshot?.businessInfo?.legalType || null)
 
     // store Business Information
-    // *** FUTURE: fix this
-    this.setBusinessInformation({ ...entitySnapshot?.businessInfo } || { ...EmptyBusinessInfo })
```

Comment:
Error was, "first express is always True", since it's object (maybe empty).

The new code is smarter and correct.